english/super-man/save-the-word.md

```diff
@@ -1,6 +1,7 @@
 After that, suddenly!!! You start to grow up, Big, GIANT ~~~ 
 Also you noticed that a red shirt is now wrapped on your muscular body.
-The trangle underwear is mounted on it, shining a great "V" 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.